config/config/sc-config.yaml

```diff
@@ -115,10 +115,10 @@ grafana:
       resources:
         requests:
           cpu: 10m
-          memory: 80Mi
+          memory: 150Mi
         limits:
-          cpu: 50m
-          memory: 100Mi
+          cpu: 250m
+          memory: 200Mi
```

Comment:
Is there a reason for this change as well? Like will it not start if this is set to the previous value or is it just unusably slow?

Reply:
Yeah, it gets super slow.. Then we had a customer situation that made us aware that we may need a limit at 250m.